packages/calcite-components/src/components/list-item-group/list-item-group.tsx

```diff
@@ -36,6 +37,9 @@ export class ListItemGroup extends LitElement implements InteractiveComponent {
   /** The header text for all nested `calcite-list-item` rows. */
   @property({ reflect: true }) heading: string;
 
+  /** Specifies the size of the component. */
+  @property({ reflect: true }) scale: Scale = "m";
```

Comment:
Can we make these internal and have `list` set them directly?

Reply:
For internal properties, should we try to not reflect them? It more easily exposes them to the user and lets them see internal attr values. Maybe this is a discussion we should have @jcfranco? At least when they are not reflected a user isn't seeing them in the DOM tree.

Reply:
I opted to use attribute selectors for the styling to get this done as fast as possible, so if we feel strongly enough about not reflecting internal attributes we would have to refactor the styling to use css classes instead.

Reply:
Personally I don't have qualms about reflecting internal attributes as long as they're meant to be reflected for styling purposes and also are decorated with `@internal ` or `@private`.